src/gui_common/tooltip/ToolTipManager.cs

```diff
@@ -516,4 +519,54 @@ private void HideAllToolTips()
             }
         }
     }
+
+    private void CreateOrganelleTooltips()
+    {
+        var packedTooltip = GD.Load<PackedScene>("res://src/microbe_stage/editor/tooltips/SelectionMenuToolTip.tscn");
+
+        foreach (var organelle in SimulationParameters.Instance.GetAllOrganelles())
+        {
+            var tooltip = packedTooltip.Instantiate<SelectionMenuToolTip>();
+
+            Instance.AddToolTip(tooltip, "organelleSelection");
+
+            tooltip.Description = organelle.Description;
+            tooltip.MutationPointCost = organelle.MPCost;
+            tooltip.Name = organelle.InternalName;
+            tooltip.DisplayName = organelle.Name;
+            tooltip.RequiresNucleus = organelle.RequiresNucleus;
+            tooltip.ThriveopediaPageName = organelle.InternalName;
+            tooltip.ProcessesDescription = organelle.ProcessesDescription == null ?
+                string.Empty :
+                organelle.ProcessesDescription;
+
+            if (organelle.Components.Storage != null)
+            {
+                tooltip.AddModifierInfo(new LocalizedString("STORAGE").ToString(),
+                    "+" + organelle.Components.Storage.Capacity.ToString(CultureInfo.CurrentCulture), 0,
+                    "res://assets/textures/gui/bevel/StorageIcon.png");
+            }
+
+            if (organelle.Components.Lysosome != null)
+            {
+                tooltip.AddModifierInfo(new LocalizedString("DIGESTION_SPEED").ToString(),
+                    "+" + (Constants.ENGULF_COMPOUND_ABSORBING_PER_SECOND *
+                        Constants.ENZYME_DIGESTION_SPEED_UP_FRACTION).ToString(CultureInfo.CurrentCulture), 0,
```

Comment:
Using current culture text formatting here, means that this data has to be re-generated when the selected game language changes.